modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/actions/PreIssueAccessTokenClientCredentialsGrantTestCase.java

```diff
@@ -322,7 +379,95 @@ public void testTokenExpiresInClaimReplaceOperation() throws Exception {
         Date iat = jwtClaims.getDateClaim("iat");
         long expiresIn = (exp.getTime() - iat.getTime()) / 1000;
 
-        Assert.assertEquals(expiresIn, 7200);
+        Assert.assertEquals(expiresIn, UPDATED_EXPIRY_TIME_PERIOD);
+    }
+
+    /**
+     * Retrieves pre issue access token action request.
+     *
+     * @return pre issue access token request object
+     */
+    private PreIssueAccessTokenActionRequest getRequest() {
+
+        TokenRequest tokenRequest = createTokenRequest();
+        AccessToken accessTokenInRequest = createAccessToken();
+
+        Tenant tenant = new Tenant(tenantId, tenantInfo.getDomain());
+        User user = new User(userId);
+        UserStore userStore =
+                new UserStore(Base64.getEncoder().encodeToString("PRIMARY".getBytes(StandardCharsets.UTF_8)),
+                        "PRIMARY");
+
+        PreIssueAccessTokenEvent event = new PreIssueAccessTokenEvent.Builder()
+                .request(tokenRequest)
+                .accessToken(accessTokenInRequest)
+                .tenant(tenant)
+                .organization(null)
+                .user(user)
```

Comment:
For client credentials ideally there should not be a user 
Is there a user in the request ?